index.js

```diff
@@ -1,6 +1,5 @@
 const jsExtensions = ['js', 'cjs', 'mjs', 'jsx'];
 const tsExtensions = ['ts', 'cts', 'mts', 'tsx'];
-const allExtensions = [...jsExtensions, ...tsExtensions];
```

Comment:
This was removed because it's not used any more. Do we want to export the list of extensions? We could do that from `eslint-config-seek`.